src/riot-rs-macros/src/thread.rs

```diff
@@ -22,7 +22,11 @@
 /// This starts a thread with a stack size of 1024 bytes and a priority of 2:
 ///
 /// ```ignore
-/// #[riot_rs::thread(stacksize = 1024, priority = 2)]
+/// const STACK_SIZE: usize = 1024;
```

Comment:
There's no need for the constants here, so let's have the example succinct. Maybe either remove changes? Alternatively, change only one of the literals to an expression and add an inline comment like `expressions can also be used`?

Reply:
great, thanks!